mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -648,6 +668,7 @@ def create_model_version(
                     tags=uc_model_version_tag_from_mlflow_tags(tags),
                     run_tracking_server_id=source_workspace_id,
                     feature_deps=feature_deps,
+                    model_version_dependencies = other_model_deps
```

Comment:
@victorsun123 Can we write some unit tests to make sure that we're populating this field correctly?

Reply:
+1